detection-rules/attachment_html_recipient_in_javascript_identifiers.yml

```diff
@@ -16,13 +16,18 @@ source: |
             )
             or .file_extension in~ $file_extensions_common_archives
             or .file_type == "html"
+            or .content_type == "text/html"
           )
           and any(file.explode(.),
                   // suspicious identifiers
-                  any(.scan.javascript.identifiers, strings.like(., "atob", "decrypt"))
+                  any([.scan.strings.strings, .scan.javascript.identifiers],
+                      any(., strings.like(., "*atob*", "*decrypt*"))
+                  )
                   // Recipients address found in javascript
-                  and any(recipients.to,
-                          any(..scan.javascript.strings, strings.icontains(., ..email.email))
+                  and any(file.explode(..),
```

Comment:
this is nested inside the other fileexplode is that intentional?

Reply:
yes. this rule should be firing all the time, but in every sample where the condition is true, it will not fire without the double file explode, since the recipients email is found in a different exploded piece resulting in something like this 

![image](https://github.com/sublime-security/sublime-rules/assets/6809735/7528747b-6410-4b4f-9e42-c0d3207c2d1f)

Reply:
you still should be able to do it without the file explode inside a file explode